test/pom.xml

```diff
@@ -53,7 +53,7 @@ THE SOFTWARE.
     <dependency>
       <groupId>${project.groupId}</groupId>
       <artifactId>jenkins-test-harness</artifactId>
-      <version>2.31</version>
+      <version>2.32</version>
```

Comment:
would be better to use a dependencyManagement

Reply:
Why? There is exactly one dependency on this artifact from this repository.

Reply:
Do not recall why I said that